js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetLiveDataProvider.tsx

```diff
@@ -109,132 +146,24 @@ export const AssetLiveDataProvider = ({children}: {children: React.ReactNode}) =
             (e.stepKey && assetStepKeys.has(e.stepKey)),
         )
       ) {
-        factory.manager.invalidateCache();
+        AssetBaseData.manager.invalidateCache();
```

Comment:
Does this need the second call to invalidateCache as well?

Reply:
Oops yeah

Reply:
oh wait it doesn't!